atlp-rwanda/phantom-be-dream-team
# #15 Feature: View a list of buses

#### What does this PR do?

- Implements View a list of buses

#### Description of Task to be completed?

- As a user, I want to choose origin & destination locations so that I can see a list of buses going from the origin location to the destination location.
- User have to see an ordered list of buses available 


#### How should this be manually tested?

- Clone the repo

- checkout to branch `ft-view-list-of-buses-#15`

- Run `npm install` to install modules

- Run `npm run dev:start` to start the development server

- Run `npm run test` to run tests

- In your API testing tool (postman recommended) call api `localhost:port/
- localhost:port/

1. **GET: api/v1/listbuses/allbuses/from/to**   to see all buses 

**NOTE:** 

- replace the **from** with the origin place of the user
- replace the **to** with the destination place  of the user
- replace the **port** with the port you are listening from your computer

#### What are the relevant Pivotal Tracker/trello stories?

- link to my trello https://trello.com/c/rgPfFs6P/15-view-a-list-of-buses

#### Screenshots 

- 
#### Questions:

- N/A
